editor/blocks/quote-block/index.js

```diff
@@ -0,0 +1,23 @@
+const { query, html, text } = wp.blocks.query;
+
+wp.blocks.registerBlock( 'wp/quote', {
+	title: 'Quote',
+	icon: 'format-quote',
+
+	attributes: {
+		value: query( 'blockquote', html( 'p' ) ),
+		citation: query( 'cite,footer', text() )
```

Comment:
Also on the cite vs footer issue, #311 uses cite, `post-content.js` uses footer, so I put in both for the time being.  We will want to make a decision one way or the other and I believe mtias has already made that decision in #311, so maybe we should only parse cite.